a10_neutron_lbaas/db/models.py

```diff
@@ -53,24 +59,47 @@ def find_all(cls, db_session=None):
         return cls.query(db_session).all()
 
     @classmethod
+    def create(cls, **kwargs):
```

Comment:
You can convert the mixin to not-a-mizin by extending A10Base and add `__abstract__ = True`